draft-packages/loading-spinner/index.ts

```diff
@@ -1 +1 @@
-export * from "./KaizenDraft/LoadingSpinner/LoadingSpinner"
+export { default as LoadingSpinner } from "./KaizenDraft/LoadingSpinner/LoadingSpinner"
```

Comment:
This no longer exports the props. Would we be able to just directly place the HOC around the component in L17? Although it's possible it might be less readable doing that, in which case we will need to add the props export in here.
Alt, rename the component itself (`LoadingSpinnerComponent`?) which you pass into the HOC and have that be exported as `LoadingSpinner`.

Reply:
I might opt for the second options for readability, I tried the first one but it wasn't the prettiest when it came to readability